lib/helpers/validateSignupData.js

```diff
@@ -0,0 +1,32 @@
+import Schema from 'validate'
+
+const validateSignupData = data => {
+  const userData = new Schema({
```

Comment:
You can take the `Schema` definition out of the function. this will fix the CodeClimate issue.